src/main/java/com/vmware/vipclient/i18n/util/LocaleUtility.java

```diff
@@ -6,15 +6,20 @@
 
 import java.util.Arrays;
 import java.util.HashMap;
+import java.util.LinkedList;
 import java.util.List;
 import java.util.Locale;
 import java.util.Map;
 import java.util.stream.Collectors;
 
 public class LocaleUtility {
 
-    private static Locale defaultLocale  = Locale.ENGLISH;
-
+    private static Locale defaultLocale = Locale.ENGLISH;
+    private static Locale sourceLocale;
```

Comment:
It is handled in the getter so that if the default locale is set to something else, the source locale will get it:
`public static Locale getSourceLocale() {
		return sourceLocale == null ? getDefaultLocale() : sourceLocale; 
	}`

Reply:
Looks like you get defaultLocale and sourceLocale opposite.
I think it should be: By default the sourceLocale=ENGLISH, if it's set to non-ENGLISH locale then use the config's value; if defaultLocale is not set, use sourceLocale as the defaultLocale, else use the config's defaultLocale.

Reply:
makes sense